include/sys/mimiker.h

```diff
@@ -15,7 +15,7 @@
 #include <sys/param.h>
 #include <sys/types.h>
 
-#define log2(x) (__builtin_ffs(x) - 1)
+#define log2(x) (31 - __builtin_clz(x))
```

Comment:
I'm afraid it's not going to work on _AArch64_ :(

Reply:
Well in fact... it won't work with `int16_t` nor `int8_t`...

Reply:
> Well in fact... it won't work with int16_t nor int8_t...

It does work with shorter data types than int since the builtin's prototype is in fact `int __builtin_clz (unsigned int x)`. It converts the argument to unsigned int.